web/src/features/fba/slices/runDatesSlice.test.ts

```diff
@@ -0,0 +1,196 @@
+import { DateTime } from 'luxon'
+import { beforeEach, describe, expect, it, vi } from 'vitest'
+
+// ✅ Adjust the path below to where your slice file lives
+import reducer, {
+  fetchSFMSBounds,
+  fetchSFMSRunDates,
+  getRunDatesFailed,
+  getRunDatesStart,
+  getRunDatesSuccess,
+  getSFMSBoundsFailed,
+  getSFMSBoundsStart,
+  getSFMSBoundsSuccess,
+  initialState
+} from '@/features/fba/slices/runDatesSlice'
+
+// We mock the API and error logger while keeping TS types
+vi.mock('@/api/fbaAPI', async () => {
```

Comment:
Should this be in the root describe block? We usually don't have code above imports.